src/parse_headers.cpp

```diff
@@ -1,31 +1,31 @@
-#include <Rcpp.h>
+#include <vector>
 #include <string>
 #include <sstream>
-#include "types.h"
+#include <unordered_map>
+#include <Rcpp.h>
 #include "utils.h"
 
+using map = std::unordered_map<std::string, std::vector<std::string>>;
```

Comment:
can easily confuse with `std::map`, better to find another name

Reply:
This alias used only in this translation unit and `std` namespace not used here.
So I can simply move it to the function body.